### PR TITLE
Added Dutch Translation

### PR DESCRIPTION
**Pull request by beJels**
I have added the Dutch translation.
FYI: Dutch is spoken in the Netherlands & in the largest part of Belgium (Flanders).
https://en.wikipedia.org/wiki/Flemish_Region

### CODE REVIEW

**Comment by ThierryOllivero** [comment]
@beJels : OK on my side for the changes I requested. Could you look at the changes requested by Oli for the server.js file? 
Then we can merge :)

T.

File: web/locales/nl.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    },
+    "tooltips": {
+        "electricityprice": "Elektriciteitsprijs (day-ahead)",
+        "fossilfuels": "fossiele brandstoffen",
+        "crossborderexport": "uitvoer over de landsgrenzen",
+        "carbonintensityexport": "Koolstof intensiteit van uitvoer",
+        "of": "van de",
+        "ofconsumption": "van consumptie",
+        "ofinstalled": "van de geïnstalleerde capaciteit",
+        "ofelectricity": "",
```
Comment (corradio) [inline web/locales/nl.json]: Missing translation

**Comment by corradio** [comment]
@beJels can you carefully look at the screenshots and make sure everything is ok?
There's one translation missing also.

image
image
image
image
image
image
image

File: web/locales/nl.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    },
+    "country-panel": {
+        "back": "terug",
+        "source": "bron",
+        "fossilfuels": "fossiele brandstoffen",
+        "electricityprice": "Elektriciteitsprijs (day-ahead)",
+        "electricityproduction": "Elektriciteitsproductie",
+        "showemissions": "toon uitstoot",
+        "bysource": "per bron",
+        "emissions": "uitstoot",
```
Comment (corradio) [inline web/locales/nl.json]: Requires capitalization

**Comment by corradio** [comment]
@beJels I'm going to merge and do the last required changes. Please comment here if there's something wrong with the screenshots and I'll change it!

File: web/locales/nl.json (hunk 1 above)
Comment (beJels) [inline web/locales/nl.json]: The missing translation should be:
 "ofelectricity": "van de",

**Comment by beJels** [comment]
You might want to replace this "past tense" of the verb namely:"Ondervond" in
       "foundbugs": "Ondervond je problemen of heb je ideëen? Meld ze",
by the present tense form of the verb "Ondervind" in
       "foundbugs": "Ondervind je problemen of heb je ideëen? Meld ze",
Further the screenshots all look fine.

**Comment by corradio** [comment]
I'll change it if you believe it sounds better with the present tense 👍

**Pull request merged**